back/src/controllers/userController.js

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable quotes */
+import { userService } from "../services/userService.js";
+
+async function getAll(req, res, next) {
+  const users = await userService.getAllActive();
+
+  res.send(users.map(userService.normalize));
+}
+
+async function updateName(req, res, next) {
+  try {
+    const { email, fullName } = req.body;
+
+    await userService.updateName(email, fullName);
+
+    res.send({ status: "OK" });
```

Comment:
```suggestion
    res.status(200).end()
```

Reply:
Fixed. Instead of end object with message in send method was used.